src/build.sh

```diff
@@ -151,6 +162,20 @@ function parse_build_options()
         options_values['MENU_CONFIG']="${build_config[menu_config]:-nconfig}"
         shift
         ;;
+      --cpu-util | --cpu-scale | -c)
```

Comment:
Why do we want two names here?
Also, you forgot to update the help function and the man page.

Reply:
I'm leaving docs for last, sorry about that.